### ISSUE TITLE
Edited workshop submarine was erased

### ISSUE THREAD

**Issue by Diffle** [Regalis11/Barotrauma#2748]
**Description**
I edited workshop sub, saved it with another name, and when sub on workshop was updated it just erased. It's just dissapeared in Mods/*SubmarineName* folder.

**Steps To Reproduce**
Edit workshop sub, save it with another name, and wait for original submarine update.

**Version**
v0.9.9.1, Windows, revision 1968ab79f

**Comment by juanjp600** [comment]
I suspect this issue is different from #2570. When saving a submarine that belongs to a content package that you haven't published, it'll still save in that content package's directory. When that mod is updated, it will (correctly) clear the folder before installing the new files.

This case should be fixed by forcing submarines to be saved in the Submarines folder if you don't own the content package it belongs to: https://github.com/Regalis11/Barotrauma-development/commit/23cc2e5d8071272ee6ad37ec869cd2bd239be74a

**Comment by Rokvach** [comment]
Found a issue, if you delete the original published sub, the copy of that sub you created gets turned in to a mod which is disabled by default, which makes it look as if the sub was entirely deleted as it isn't in the submarines folder anymore and can't be seen in the submarines list. 

How I think this should work is if you create a copy of a published submarine that copy should always stay in the submarines folder until you publish it yourself. @juanjp600 

How I tested this:
1. Created a new version of Dugong, saved it with a different name Dugong 2
2. Published to workshop
3. Opened Dugong 2 on editor
4. Saved with a different name Dugong 3
5. Updated Dugong 2 on workshop
6. Opened editor and checked that Dugong 3 still exists and checked folder structure (Dugong 3 was saved to /Submarines successfully)
8. Removed Dugong 2 using sub editor
9. Noticed that Dugong 3 was moved to the /Mods folder and is disabled in settings
